src/Sulu/Bundle/SecurityBundle/Command/CreateUserCommand.php

```diff
@@ -35,69 +35,23 @@ class CreateUserCommand extends Command
 {
     protected static $defaultName = 'sulu:security:user:create';
 
-    /**
-     * @var EntityManagerInterface
-     */
-    private $entityManager;
-
-    /**
-     * @var UserRepository
-     */
-    private $userRepository;
-
-    /**
-     * @var RoleRepositoryInterface
-     */
-    private $roleRepository;
-
-    /**
-     * @var ContactRepositoryInterface
-     */
-    private $contactRepository;
-
-    /**
-     * @var LocalizationManagerInterface
-     */
-    private $localizationManager;
-
-    /**
-     * @var SaltGenerator
-     */
-    private $saltGenerator;
-
-    /**
-     * @var PasswordHasherFactoryInterface|EncoderFactoryInterface
-     */
-    private $passwordHasherFactory;
-
-    /**
-     * @var string[]
-     */
-    private $locales;
-
     /**
      * @param PasswordHasherFactoryInterface|EncoderFactoryInterface $passwordHasherFactory
      */
     public function __construct(
-        EntityManagerInterface $entityManager,
-        UserRepository $userRepository,
-        RoleRepositoryInterface $roleRepository,
-        ContactRepositoryInterface $contactRepository,
-        LocalizationManagerInterface $localizationManager,
-        SaltGenerator $saltGenerator,
-        $passwordHasherFactory,
-        array $locales
+        private EntityManagerInterface $entityManager,
+        private UserRepository $userRepository,
+        private RoleRepositoryInterface $roleRepository,
+        private ContactRepositoryInterface $contactRepository,
+        private LocalizationManagerInterface $localizationManager,
+        private SaltGenerator $saltGenerator,
+        private $passwordHasherFactory,
+        /**
+         * @var string[]
+         */
```

Comment:
Would move the phpdoc up to the `@param` like we already do for the encoder

Reply:
Will do.